### PR TITLE
Recommended pkgs

### PR DESCRIPTION
**Pull request by mvidner**
https://trello.com/c/fc9pedpd
https://fate.suse.com/318099
https://bugzilla.opensuse.org/show_bug.cgi?id=900853
https://bugzilla.opensuse.org/show_bug.cgi?id=902394

Note that this implements Plan A (move from Option to Command) but https://github.com/libyui/libyui-qt-pkg/commit/d12749b4940bcbccc85f889365dfc39563c609ed can be cleanly omitted and then IRPfARP will be gone without replacement (Plan B).

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+- Unified option handling with NCurses (FATE#318099)
+  - "System verification mode" and "Dependencies/Autocheck"
+    moved to /etc/sysconfig/yast2 (PKGMGR_VERIFY_SYSTEM,
+    PKGMGR_AUTO_CHECK).
+  - made "Cleanup when deleting packages" and "Allow vendor
+    change" not persistent.
+- Added option Dependencies/Install Recommended Packages,
+  (PKGMGR_RECOMMENDED) (boo#900853).
+- Changed "Options/Ignore Recommended Packages for Already
+  Installed Packages" from an option to a command
```
Comment (kobliha) [inline ChangeLog]: from an option to a command? I thought we wanted this to be an option
Command should have been: Install Recommended Packages (now)

File: ChangeLog [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+Mon Jun 22 13:46:17 UTC 2015 - mvidner@suse.com
+
+- Unified option handling with NCurses (FATE#318099)
+  - "System verification mode" and "Dependencies/Autocheck"
+    moved to /etc/sysconfig/yast2 (PKGMGR_VERIFY_SYSTEM,
+    PKGMGR_AUTO_CHECK).
+  - made "Cleanup when deleting packages" and "Allow vendor
+    change" not persistent.
+- Added option Dependencies/Install Recommended Packages,
+  (PKGMGR_RECOMMENDED) (boo#900853).
```
Comment (kobliha) [inline ChangeLog]: I thought we wanted this to be a command, not option.

File: ChangeLog [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+- Changed "Options/Ignore Recommended Packages for Already
+  Installed Packages" from an option to a command
+  "Extras/Install All Matching Recommended Packages" (boo#902394).
```
Comment (kobliha) [inline ChangeLog]: Like this, this is a `command` if I'm not mistaken

File: ChangeLog [hunk 4]
```diff
@@ -1,4 +1,20 @@
 -------------------------------------------------------------------
+Mon Jun 22 13:46:17 UTC 2015 - mvidner@suse.com
+
+- Unified option handling with NCurses (FATE#318099)
+  - "System verification mode" and "Dependencies/Autocheck"
+    moved to /etc/sysconfig/yast2 (PKGMGR_VERIFY_SYSTEM,
+    PKGMGR_AUTO_CHECK).
+  - made "Cleanup when deleting packages" and "Allow vendor
+    change" not persistent.
```
Comment (kobliha) [inline ChangeLog]: Was this actually required?

**Comment by shundhammer** [comment]
Looks good to me.

File: ChangeLog (hunk 2 above)
Comment (kobliha) [inline ChangeLog]: OK

File: ChangeLog (hunk 3 above)
Comment (kobliha) [inline ChangeLog]: OK, I've parsed this text in a wrong way last time

File: ChangeLog (hunk 4 above)
Comment (kobliha) [inline ChangeLog]: And why is this not persistent? This is UI, not commandline, where users expect default behavior (or changed via option). Where can I find any complaint that Yast keeps Vendor Change persistent?

Comment (mvidner) [inline ChangeLog]: To make it consistent with libyui-ncurses-pkg, where it is considered a risky option to be used only when needed and not left enabled for next runs when you forget about it.

**Pull request merged**